### PR TITLE
Textual changes to the OLO Flow

### PR DESCRIPTION
**Pull request by svenjens**
Description 
This PR takes care of some text changes in the olo flow.
We also removed the zonningplan information and all the test for zoningplans, because we are not using it anymore. 

 
Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [x] you followed our checklist for [functional testing](https://github.com/Amsterdam/vergunningcheck/blob/master/TESTING.md)
- [ ] you added the necessary documentation (either in the markdown files or inline)
- [x] you added the necessary automated tests

### CODE REVIEW

File: packages/client/src/intros/shared/DefaultOloIntro.js [hunk 1]
```diff
@@ -1,21 +1,10 @@
...
-      <ListItem>U voert eerst het adres van het gebouw in.</ListItem>
-      <ListItem>
-        Vervolgens krijgt u informatie over het gebouw te zien.
-      </ListItem>
-      <ListItem>
-        U gebruikt deze informatie om de vergunningcheck te doen op het
-        Omgevingsloket.
-      </ListItem>
-    </List>
+    <Paragraph>U voert eerst het adres van het gebouw in</Paragraph>
```
Comment (robinpiets) [inline packages/client/src/intros/shared/DefaultOloIntro.js]: This text is not according to the renewed texts

**Review by robinpiets — CHANGES_REQUESTED** [review]
Intro text is not updated

**Comment by afjlambert** [comment]
Please make sure the graphql query is updated accordingly. Please don't remove the implementation from the graphql server. Thanks.

**Review by robinpiets — APPROVED** [review]
Looks good. Let's remove the Olo Intro page in a new PR

**Pull request merged**